### ISSUE TITLE
A11y(Nav): When navigating with the keyboard there is no way to skip the nav straight to the content

### ISSUE THREAD

**Issue by JoaoSilvaGrafana** [grafana/grafana#60816]
**Steps to reproduce**:

Using only the keyboard, navigate to a different page and then use tab to navigate to the content

**Actual Result**:

Tabbing through:

https://user-images.githubusercontent.com/100691367/209965048-5f4803f1-8b67-428d-9372-c2f2fa29bc10.mov

**Expected Result**

There should be some kind of shortcut or button "go to content" that lets the user skip the whole navigation and focus on the content.

**Relevant WCAG Criteria:** [2.4.1 WCAG Criterion - Bypass Blocks](https://www.w3.org/WAI/WCAG21/quickref/#bypass-blocks)

**Environment**:
- Grafana version: any
- Data source type & version: -
- User OS & Browser: -
- Others: With topnav turned on

### Severity
`SEV2`
